joint_state_publisher_gui:
  ros__parameters:
    joint_1:
      min: -2.967060
      max:  2.967060
    joint_2:
      min: -1.570796
      max:  2.356194
    joint_3:
      min: -1.396263
      max:  2.932153
    joint_4:
      min: -3.228859
      max:  3.228859
    joint_5:
      min: -2.094395
      max:  2.094395
    joint_6:
      min: -6.283185
      max:  6.283185

